lib/ruby_smb/ntlm/custom/ntlm.rb

```diff
@@ -0,0 +1,19 @@
+require 'net/ntlm'
```

Comment:
To ensure this is loaded whenever `ruby_smb` is it might be worth adding a `require 'custom/ntlm'` [here](https://github.com/rapid7/ruby_smb/blob/374ea4602956a47c415bab5353979ea0ba421768/lib/ruby_smb.rb#L8).

I am not sold on using `Custom` as the top level namespace in a `gem`, but I don't have a better option in mind so just noting this.

Reply:
Thanks! Addressed in [10997a3](https://github.com/rapid7/ruby_smb/pull/238/commits/10997a351ffaf9e34d9522ef63d823f5fffc0a85)